### PR TITLE
feat: DEV-2717: Audio Controls

### PR DESCRIPTION
**Pull request by juliosgarbi**
New UI for audio controls

### CODE REVIEW

File: src/components/Timeline/Controls.tsx [hunk 1]
```diff
@@ -219,14 +281,25 @@ export const Controls: FC<TimelineControlsProps> = memo(({
...
+        ) : (
+          <TimeDurationControl
+            startTime={0}
+            endTime={duration}
+            minTime={0}
+            maxTime={duration}
+            currentTime={currentTime}
+            onChangeStartTime={onTimeUpdateChange}
+          />
+        )}
```
Comment (bmartel) [inline src/components/Timeline/Controls.tsx:302]: Probably should swap this around to being in the positive tense as it reads a bit easier.
```suggestion
        {isFF(FF_DEV_2715) ? (
          <TimeDurationControl
            startTime={0}
            endTime={duration}
            minTime={0}
            maxTime={duration}
            currentTime={currentTime}
            onChangeStartTime={onTimeUpdateChange}
          />
        ) : (
          <TimeDisplay
            currentTime={currentTime}
            duration={duration}
            length={length}
            position={position}
            framerate={frameRate}
            formatPosition={formatPosition}
          />
        )}
```

File: src/components/Timeline/Controls.tsx [hunk 2]
```diff
@@ -61,6 +83,40 @@ export const Controls: FC<TimelineControlsProps> = memo(({
...
+          speed={props.speed || 0}
+          zoom={props.zoom || 0}
+        />
+        <AudioControl
+          volume={props.volume || 0}
+          onVolumeChange={props.onVolumeChange}
+          onSetModal={onSetVolumeModal}
+          audioModal={audioModal}
+        />
+
```
Comment (bmartel) [inline src/components/Timeline/Controls.tsx:115]:
```suggestion
```

**Review by bmartel — CHANGES_REQUESTED** [review]
Looks good, couple changes to be made. Also I am thinking when we go to integrate the new lib with this work, we are going to want to change how we render controls. The audio information can be spammy, so we want to make double sure we aren't causing needless re-renders in react components. (probably implement some memoization and useCallbacks etc.)

**Comment by juliosgarbi** [comment]
it's already in fb-dev-2715

**Pull request closed without merging**